### ISSUE TITLE
Create project in Android studio

### ISSUE THREAD

**Issue by DevArenaCN** [SE17GroupH/Zap_fingerprint#3]

**Comment by thegreyd** [comment]
@DevArenaCN We haven't decided what and how the app is going to work? We could have design mockups of the app. The app we forked does recognition of the fingerprint (which is already stored on the phone. It does not allow to configure other than what is already recognized), before showing a new activity screen. We could add a username field to that(or not because there is only one password which is the one stored on the phone).

**Comment by thegreyd** [comment]
The forked repository has all the necessary project files. we can import the project in android studio. I don't think we should start from scratch.

**Comment by thegreyd** [comment]
If we are only doing a demo of fingerprint authentication, without any communication to the server, then our work is really not much.

**Comment by KaustubhG** [comment]
I think we should meet tomorrow to chalk out the details. Meanwhile we can get accustomed to Android Studio environment.

**Comment by DevArenaCN** [comment]
@thegreyd Ah, cool. Didn't think of that.

**Comment by thegreyd** [comment]
Yup I'm ready to meet tomorrow before class, and maybe talk to the professor. We should think about our demo, what we are going to show, and if we potentially could have server side communication to authenticate.
